MediaBrowser.Model/Lyrics/LyricLine.cs

```diff
@@ -25,4 +29,9 @@ public LyricLine(string text, long? start = null)
     /// Gets the start time in ticks.
     /// </summary>
     public long? Start { get; }
+
+    /// <summary>
+    /// Gets the time-aligned cues for the song's lyrics.
+    /// </summary>
+    public IReadOnlyList<LyricLineCue>? Cues { get; }
```

Comment:
Make sure to always set this value (so empty list by default), but keep the nullability for backwards compatibility. Then we can make this not-null in a later API version.

Reply:
I've changed the code later to not include "null" for this object type